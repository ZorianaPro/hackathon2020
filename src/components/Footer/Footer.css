.Footer {
    text-align: center;
    margin: 2em;
}