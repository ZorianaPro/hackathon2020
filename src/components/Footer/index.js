import React from 'react';
import './Footer.css';

const Footer = () => (
  <div className="Footer">
    <div className="Figure-1"/>
    <div className="Figure-2"/>
    <div className="Figure-3"/>
    <div className="Footer-Container">
      <p className="Footer-Title">
        Credits
      </p>
      <div className="Credits">
        <div className="Credits-Person">
          <div className="Credits-Person-Photo">
            <img src="/assets/zl.png" alt=""/>
          </div>
          <p className="Credits-Person-Name">
            Zoriana Lesyk
          </p>
          <p className="Credits-Person-Position">
            Project lead & Front end development
          </p>
        </div>
        <div className="Credits-Person">
          <div className="Credits-Person-Photo">
            <img src="/assets/yd.png" alt=""/>
          </div>
          <p className="Credits-Person-Name">
            Yoana Dimova
          </p>
          <p className="Credits-Person-Position">
            Project lead & Front end development
          </p>
        </div>
        <div className="Credits-Person">
          <div className="Credits-Person-Photo">
            <img src="/assets/sy.png" alt=""/>
          </div>
          <p className="Credits-Person-Name">
            Sebastian Yepes
          </p>
          <p className="Credits-Person-Position">
            Illustrations, Layout & UI
          </p>
        </div>
        <div className="Credits-Person">
          <div className="Credits-Person-Photo">
            <img src="/assets/hp.png" alt=""/>
          </div>
          <p className="Credits-Person-Name">
             Hans-Peter Dietz
          </p>
          <p className="Credits-Person-Position">
            Front end support
          </p>
        </div>
        <div className="Credits-Person">
          <div className="Credits-Person-Photo">
            <img src="/assets/vk.png" alt=""/>
          </div>
          <p className="Credits-Person-Name">
            Valentin Konov
          </p>
          <p className="Credits-Person-Position">
            Deployment
          </p>
        </div>
        <div className="Credits-Person">
          <div className="Credits-Person-Photo">
            <img src="/assets/ar.jpg" alt=""/>
          </div>
          <p className="Credits-Person-Name">
            Ad√£o Raul
          </p>
          <p className="Credits-Person-Position">
            DevOps
          </p>
        </div>
      </div>
    </div>
  </div>
);

export default Footer;
