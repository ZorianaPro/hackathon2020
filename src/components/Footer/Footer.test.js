import React from 'react';
import { shallow } from "enzyme";
import Footer from './';

describe('Footer', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <Footer/>)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<Footer/>)).toMatchSnapshot();
  });
});

