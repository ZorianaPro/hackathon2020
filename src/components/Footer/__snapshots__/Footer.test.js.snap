// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as expected 1`] = `
<div
  className="Footer"
>
  2020 @ All right reserved
</div>
`;
